service_container/factories.rst

```diff
@@ -300,12 +300,23 @@ Usage example
 The following example is intended to show how to create and use a factory method in Symfony framework.
 Suppose you want to realize the factory method pattern for services, that describe two delivery methods - DHL and UPS.
 
+Firstly, define marker interface for delivery methods classes::
+
+    // src/Deliveries/DeliveryInterface.php
+    namespace App\Deliveries;
+	
+    interface DeliveryInterface
+    {
+    }
+
```

Comment:
This can be removed completely

Reply:
Let me leave this until we reach a consensus on the use of the marker interface.